spec/spec_helper.rb

```diff
@@ -1,95 +1,12 @@
-# Run Coverage report
-require 'coveralls'
-Coveralls.wear!('rails')
-
 # Configure Rails Environment
 ENV['RAILS_ENV'] = 'test'
 
-begin
-  require File.expand_path('../dummy/config/environment.rb', __FILE__)
-rescue LoadError
-  puts 'Could not load dummy application. Please ensure you have run `bundle exec rake test_app`'
-  exit
-end
-
-require 'rspec/rails'
-require 'database_cleaner'
-require 'ffaker'
-
-# Requires supporting ruby files with custom matchers and macros, etc,
-# in spec/support/ and its subdirectories.
-Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
+require File.expand_path('../dummy/config/environment.rb', __FILE__)
 
-# Allows Sidekiq to be configured for testing
-require 'sidekiq/testing'
+require 'spree_dev_tools/rspec/spec_helper'
 
-# Requires factories defined in spree_core
-require 'spree/testing_support/factories'
-require 'spree/testing_support/controller_requests'
-require 'spree/testing_support/authorization_helpers'
-require 'spree/testing_support/url_helpers'
-
-# Requires factories defined in lib/spree_products_qa/factories.rb
 require 'spree_products_qa/factories'
 
-RSpec.configure do |config|
-  config.include FactoryGirl::Syntax::Methods
-  config.include Devise::TestHelpers, :type => :controller
-  config.include Spree::TestingSupport::ControllerRequests, type: :controller
-
-  # == URL Helpers
-  #
-  # Allows access to Spree's routes in specs:
-  #
-  # visit spree.admin_path
-  # current_path.should eql(spree.products_path)
-  config.include Spree::TestingSupport::UrlHelpers
-
-  # == Mock Framework
-  #
-  # If you prefer to use mocha, flexmock or RR, uncomment the appropriate line:
-  #
-  # config.mock_with :mocha
-  # config.mock_with :flexmock
-  # config.mock_with :rr
-  config.mock_with :rspec
-  config.color = true
-
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
-
-  # Capybara javascript drivers require transactional fixtures set to false, and we use DatabaseCleaner
-  # to cleanup after each test instead.  Without transactional fixtures set to false the records created
-  # to setup a test will be unavailable to the browser, which runs under a seperate server instance.
-  config.use_transactional_fixtures = false
-
-  # Ensure Suite is set to use transactions for speed.
-  config.before :suite do
-    DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with :truncation
-  end
-
-  # Before each spec check if it is a Javascript test and switch between using database transactions or not where necessary.
-  config.before :each do |example|
-    DatabaseCleaner.strategy = example.metadata[:js] ? :truncation : :transaction
-    DatabaseCleaner.start
-
-    Sidekiq::Worker.clear_all
```

Comment:
this gem has been using worker only for mailer... lets explore if we can just move on to active mailer

[Very High Priority]